assets/src/domain/eventEditor/services/apollo/queries/prices/test/useFetchPrices.test.ts

```diff
@@ -38,7 +38,6 @@ describe('useFetchPrices', () => {
 
 		await waitForNextUpdate({ timeout }); // wait for response
 
-		expect(result.current.error).toBeDefined();
```

Comment:
locally I was getting error because of this line, weird

Reply:
@manzoorwanijk any ideas about this?